pkg/neotest/basic.go

```diff
@@ -41,11 +42,12 @@ func NewExecutor(t testing.TB, bc *core.Blockchain, validator, committee Signer)
 	checkMultiSigner(t, committee)
 
 	return &Executor{
-		Chain:         bc,
-		Validator:     validator,
-		Committee:     committee,
-		CommitteeHash: committee.ScriptHash(),
-		Contracts:     make(map[string]*Contract),
+		Chain:           bc,
+		Validator:       validator,
+		Committee:       committee,
+		CommitteeHash:   committee.ScriptHash(),
+		Contracts:       make(map[string]*Contract),
+		collectCoverage: isCoverageEnabled(),
```

Comment:
Please, adjust documentation of this method and write a couple of words about coverage.